### DISCUSSION TITLE
New way to preform Corkey

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1215] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I discovered this when I was trying to do the SWAB exploit, If you remove the data from your profile, (aka clicking the arrow and clicking on remove profile) when you have no internet, it has the same effect as powerwashing for me, and you can preform Corkey from there. I think it's that I cant upload or save the data when it has no internet. Essentially, I have not tested it yet but I guess this can probably be preformed consecutively, so it's just a more consistent version of #1042.

**Comment by Quartinal** [comment]
@Ant643backup2, as the creator of SWAB (as you probably already can see), there's a few things I'd like to share with you.

**#1** - SWAB doesn't work for me (anymore), so I can't exactly perform this exploit. The part that doesn't work anymore for me is the pressing of the back button after clicking "Sign in as an existing user"; all it does is put me back in the "Network not available" page on the OOBE.
**#2** - If you don't already know this, you can persistently prevent most (if not all) extensions from automatically installing by policy by setting your network's nameservers to a pre-made DNS that has a blocklist similar to [Omada](https://codeberg.org/omada/config/src/branch/main/dns/webfilters.txt)'s on the Chrome OS welcome screen. After that, all you have to do is enroll your Chromebook.

Overall, what you said is pretty interesting, and it means a lot to me to have real-world evidence that someone is using my exploit!

**Reply by ghost** [reply]
For the second thing you said, my district blocked the DNS changer, and I used to use Caudns (which is more or less the same as Omada), last year, but I think it’s patched now cuz I tried using it again and I tried using the CAUB dns editor thingy, ( I forgot what it’s called), and it doesn’t seem to work. However it does work on my HOME network, since DNS settings are unblocked. Thanks, tho.

**Reply by ghost** [reply]
Sorry I meant to say I can’t really do it at school but I can do it at home.

**Reply by Quartinal** [reply]
> For the second thing you said, my district blocked the DNS changer, and I used to use Caudns (which is more or less the same as Omada), last year, but I think it’s patched now cuz I tried using it again and I tried using the CAUB dns editor thingy, ( I forgot what it’s called), and it doesn’t seem to work. However it does work on my HOME network, since DNS settings are unblocked. Thanks, tho.

You're welcome!
